pom.xml

```diff
@@ -98,8 +98,8 @@
     <google.guava.version>31.0.1-jre</google.guava.version>
     <google.http-client.version>1.40.1</google.http-client.version>
     <google.oauth-client.version>1.32.1</google.oauth-client.version>
-    <google.protobuf.version>3.18.2</google.protobuf.version>
-    <grpc.version>1.41.1</grpc.version>
+    <google.protobuf.version>3.19.2</google.protobuf.version>
+    <grpc.version>1.43.1</grpc.version>
```

Comment:
If we are ok to upgrade these dependencies, can we backport #670 instead ? @medb

Reply:
I like to minimize changes on dependencies because it could be a blocker for dataflow to get the latest 2.2.x patch.